src/components/Bale/BaleInfo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, Text } from 'react-native';
+import strings from '../../localization';
+import styles from './styles';
+
+const BaleInfo = ({ type, weight }) => (
+  <View style={styles.containerBaleInfo}>
+    <Text style={styles.informationTitle}>{strings.information}:</Text>
+    <View style={styles.containterTypeAndWeightInfo}>
+      <View style={styles.subContainerTypeAndWeigthInfo}>
+        <Text style={styles.titleTypeAndWeightInfo}>{strings.baleComponentType}</Text>
+        <Text style={styles.subtitleTypeAndWeightInfo}>{`${type}`}</Text>
+      </View>
+      <View style={styles.subContainerTypeAndWeigthInfo}>
+        <Text style={styles.titleTypeAndWeightInfo}>{strings.baleComponentWeight}</Text>
+        <Text style={styles.subtitleTypeAndWeightInfo}>{`${weight} kg`}</Text>
```

Comment:
weight is defined as string, so you don't need to cast it to string. You can just write {weight} kg instead of {`${weight} kg`}